UIMediator/UIMediator.cpp

```diff
@@ -1188,16 +1204,29 @@ UIMediator::onOpenBookmarkManager(void)
 }
 
 void
-UIMediator::onJumpToBookmark(qint64 frequency)
+UIMediator::onJumpToBookmark(BookmarkInfo info)
 {
-  this->ui->spectrum->setCenterFreq(frequency);
+  this->ui->spectrum->setCenterFreq(info.frequency);
   this->ui->spectrum->setLoFreq(0);
 
-  this->onFrequencyChanged(frequency);
+  if (!info.modulation.isEmpty()) {
+    this->ui->audioPanel->setDemod(AudioPanel::strToDemod(info.modulation.toStdString()));
+  }
+
+  if (info.bandwidth() != 0) {
+    this->ui->spectrum->setFilterBandwidth(info.bandwidth());
```

Comment:
My bad, this actually updates the spectrum only. We should use `this->setBandwidth(unsigned int)` which is the proper mediator method that updates all UI widgets related to the filter bandwidth (including both the spectrum and the inspector panel).

Reply:
fixed and works :)